make test fails due to missing Secret model

```
ERROR collecting tests/logic/secret_test.py
ImportError while importing test module '/Users/bryanmarty/beans/tests/logic/secret_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/logic/secret_test.py:11: in <module>
    from yelp_beans.models import Secret
E   ImportError: cannot import name Secret`
```

--- a/tests/logic/secret_test.py
+++ b/tests/logic/secret_test.py
@@ -8,15 +8,6 @@ import json
 import pytest
 
 from yelp_beans.logic.secret import get_secret
-from yelp_beans.models import Secret
-
-
-def test_get_secret_database(tmpdir, database):
-    with tmpdir.as_cwd():
-        expected = 'password'
-        Secret(id='secret', value=expected).put()
-        actual = get_secret('secret')
-        assert expected == actual
 
 
 def test_get_secret_file(tmpdir, database):

